Futures-in-Scala-2.12-part-4.md

```diff
@@ -0,0 +1,31 @@
+#Futures in Scala 2.12 (part 4)
+
+This is the third of several posts describing the evolution of `scala.concurrent.Future` in Scala `2.12.x`.
+For the previous post, [click here](https://github.com/viktorklang/blog/blob/master/Futures-in-Scala-2.12-part-3.md).
```

Comment:
If I do edits I'd rather have readers see the most up to date version :) They can always browse the version history and do git blame ;)